roboteam_ai/src/control/numTrees/NumTreePosControl.h

```diff
@@ -28,16 +26,28 @@ class NumTreePosControl : public ForcePosControl {
         Vector2 finalTargetPos;
 
         bool doRecalculatePath(const Vector2 &targetPos);
-        PosVelAngle computeCommand(const Vector2 &exactTargetPos);
+        bool checkCurrentRobotCollision();
+        bool checkEmptyPath();
+        bool checkIfTargetMoved(double maxTargetDeviation, const Vector2 &targetPos);
+        bool checkIfAtEndOfPath(double maxTargetDeviation, const Vector2 &targetPos);
+        bool checkIfTooFarFromCurrentPath(double maxTargetDeviation, const Vector2 &vector2);
+
+        RobotCommand computeCommand(const Vector2 &exactTargetPos);
 
         // constants
-        const double MAX_CALCULATION_TIME = 25.0;         // Max calculation time in ms
+        const double MAX_CALCULATION_TIME = 20.0;         // Max calculation time in ms
```

Comment:
Make this in Constants.cpp

Reply:
I disagree.
these constants are really specific to numtrees, so there is no reason to do it in constants